docs/fundamentals/client/third-party-client.md

```diff
@@ -3,6 +3,16 @@
 
 This page is about accessing the Flow with other clients, e.g. `curl`, or programming languages other than python.
 
+## Golang
+
+We have developed a [Jina Go Client](https://github.com/jina-ai/client-go) that supports `grpc`, `http` and `websocket` protocol that will allow you to connect to Jina from your Go applications.
+
+
+## PHP
+
+Our community has developed a [Jina PHP Client](https://github.com/Dco-ai/php-jina)
```

Comment:
```suggestion
A big thanks to our community member [DCO.ai](https://www.dco.ai/) for developing a [PHP Client](https://github.com/Dco-ai/php-jina) for Jina!
```

Reply:
It is a company, not a person, should we name it somehow different?

Reply:
Amended